tests/api/operation_logs/test_operation_logs_permissions.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2021 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+from flask import url_for
+from invenio_accounts.testutils import login_user_via_session
+from utils import get_json
+
+from rero_ils.modules.operation_logs.api import OperationLog
+
+
+def test_operation_logs_permissions_api(
```

Comment:
missing the test functions to test `OperationLogPermission` class without using the API